<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filtro1Widget</class>
 <widget class="QWidget" name="Filtro1Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>426</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 190, 190);</string>
  </property>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson U1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson U2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson B1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson B2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson V1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson V2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson V3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson V4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson R1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson R2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Johnson R3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Halfa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale l1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton14">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale l2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton15">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Clear Window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Hot Mirror</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton17">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>UCM Halfa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton18">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Blue Prism 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Blue Prism 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton20">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Red Prism 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton21">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 13px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
	 
	background-color: rgb(213, 213, 213);
     border-style: inset;
}

QPushButton:pressed {
     background-color:blue;
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Yale Red Prism 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
